appcelerator/titanium_mobile
# [TIMOB-15785]IOS: Fixing Remote Push broken due to silent push.

Testing Instruction
1. Try sending a normal remote push while the app is in the background. - Remote push callback should fire.
2. Try build a app without include UIBackgroundModes - `fetch` or `remote-notification` in backgroundmodes.
   **Check the build apps defines.h to verify that defines.h does not contain #define USE_TI_SILENTPUSH or #define USE_TI_FETCH inside it.
3. Repeat step number 2 including the modes.  and the defines should show up.
